index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./resources/CSS/reset.css">
+  <link rel="stylesheet" href="./resources/CSS/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <header>
+    <p>My Logo</p>
```

Comment:
- [ ] Great job 👍 . Kindly wrap your logo in an `a` tag 🙏 . These are for three reasons: convention, semantics, and user experience(UX).